examples/asset_licenses.md

```diff
@@ -28,3 +28,11 @@ All assets are under public domain (CC0):
 [7]: https://www.kenney.nl/assets/ui-audio
 [8]: https://www.kenney.nl/assets/toon-characters-1
 [9]: https://www.nps.gov/subjects/sound/sounds-killdeer.htm
+
+All the following assets are under the Open Font License:
+
+| Name       | Author | Link              |
+| ---------- | ------ | ----------------- |
+| Noto fonts | Google | [Google Fonts][1] |
+
+[1]: https://fonts.google.com/noto
```

Comment:
It should be `[10]` because the link id `[1]` is already used in this document.

Reply:
Fixed.